queue_job_chunk/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Job Queue Chunk",
-    "version": "14.0.1.0.1",
+    "version": "16.0.0.0.0",
```

Comment:
Is this really the good practice, I thought the version should be restored to 16.0.1.0.0, at least, I did this in all the v16 migration I've done in OCA and nobody said anything!
Not important of course, it is just to know what is best..

Reply:
It seem to be the way oca do, so I do the same ;)